<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlicewiseEEDDialog</class>
 <widget class="QDialog" name="SlicewiseEEDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>1142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slice-wise EED Inpainting</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2dEED_inputs">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>381</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>341</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2deed_tol">
         <property name="text">
          <string>Tolerance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2deed_tol"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2deed_timeStep">
         <property name="text">
          <string>Time Step Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2deed_timeStep"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2deed_ExScheme">
       <property name="text">
        <string>Explicit Scheme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_FEDScheme">
       <property name="text">
        <string>Fast Explicit Scheme (FED)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_FSIScheme">
       <property name="text">
        <string>Fast Semi Iterative Scheme (FSI)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_mse_tol">
         <property name="text">
          <string>MSE Tolerance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_mse_tol"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_innerCycleSize">
         <property name="text">
          <string>Inner Cycle Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_innerCycleSize"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_innerLoopSIze">
         <property name="text">
          <string>Inner Loop Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_innerLoopSIze"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_loadImg">
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_mask">
       <property name="text">
        <string>Mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2dEED">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_run2dEED">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBox_monitoring">
         <property name="text">
          <string>Monitoring</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2dEED"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_imgShow">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>541</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_maskShow">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>600</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>1020</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_saveImgName">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_saveImgName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_saveImgName">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
